inc/PluginSettings/PluginSettings.php

```diff
@@ -154,19 +155,36 @@ public static function encrypt_option( array $new_value, string|array|bool $old_
 		}
 	}
 
-	public static function decrypt_option( string $value ): array|null {
+	public static function decrypt_option( string $value ): array {
```

Comment:
`decrypt_option()` was also changed to return an empty array `[]` instead of `null` on error. Returning `null` causes a runtime type failure in the parent call to [`get_config()`](https://github.com/Automattic/remote-data-blocks/blob/d73c623aa6124e86af2e8f82e93187bbb350e4da/inc/WpdbStorage/DataSourceCrud.php#L68), which expects to return an `array`.